src/aiotk/__init__.py

```diff
@@ -22,7 +25,7 @@
 from ._sched import PeriodicTask
 
 
-def run_until_complete(coro, loop=None):
+def run_until_complete(coro: Coroutine, loop: AbstractEventLoop=None):
```

Comment:
Is it actually safe to do this?  I guess if we had at least one call site, we could detect that it doesn't work because `async def()` produces `Awaitable` rather than `Coroutine` (see `EnsureDone`).